### ISSUE TITLE
HTML not using H tags

### ISSUE THREAD

**Issue by m-i-k-e** [IntersectMBO/govtool#2235]
If you inspect this page: https://sanchogov.tools/connected/governance_actions/fd348161bf87b8ff2ed0884239ba783fb5cb8659d4838bfb1e447d94fbf352d2#0

You will see that 'H' tags are not used in the structure. This is not proper semantic HTML

Here is a screenshot of the code for the top header. This should be H1:
<img width="1306" alt="image" src="https://github.com/user-attachments/assets/c7fe78f3-b48d-44e6-9909-c961fadfcca1">

As well as the H1 not being there, there are no H tags throughout the document. We need to fix this as this breaks our accessibility goals and will be impossible for people who have visual or other handicaps and rely on screenreaders to parse the page.

**Comment by MSzalowski** [comment]
That is right, part of a tech debt that we have with our current design system.

**Comment by m-i-k-e** [comment]
We discussed this in our daily meeting, and agreed that it would be ok to wait for this until we have the new design system.
@MSzalowski @bosko-m
